.jenkins/jenkins_buildbot_windows.bat

```diff
@@ -0,0 +1,56 @@
+set BUILDBOT_DIR=%WORKSPACE%\nightly_build
+set COMPILEDIR=C:\\Jenkins\\theano_cache\\buildbot_windows
+
+REM Set test reports using nosetests xunit
+set XUNIT=--with-xunit --xunit-file=
+set SUITE=--xunit-testsuite-name=
+
+set THEANO_PARAM=theano --with-timer --timer-top-n 10
+
+set THEANO_FLAGS=init_gpu_device=cuda
+
+REM Build libgpuarray
+set GPUARRAY_CONFIG="Release"
+set DEVICE=cuda
+set LIBDIR=%WORKSPACE%\local
+
+REM Make fresh clones of libgpuarray (with no history since we dont need it)
+rmdir libgpuarray /s/q
+git clone --depth 1 "https://github.com/Theano/libgpuarray.git"
+
+REM Clean up previous installs (to make sure no old files are left)
+rmdir %LIBDIR% /s/q
+mkdir %LIBDIR%
+
+REM Build libgpuarray
```

Comment:
Ok ! That's what I called "old stuff" ^^ ! But indeed, it works fine (after 2-3 tweaks also, from memory).